packages/e2e-tests/specs/splitting-merging.test.js

```diff
@@ -210,4 +210,24 @@ describe( 'splitting and merging blocks', () => {
 
 		expect( isInDefaultBlock ).toBe( true );
 	} );
+
+	it( 'should create a paragraph block above when pressing enter at start of heading', async () => {
```

Comment:
Could we move these files to `blocks/heading.test.js`, as this is specific behaviour for headings?

Reply:
That makes sense, I'll move it over.